packages/taquito/src/estimate/rpc-estimate-provider.ts

```diff
@@ -165,6 +168,8 @@ export class RPCEstimateProvider extends OperationEmitter implements EstimationP
           ? Number(result.storage_size) || 0
           : 0;
       totalStorage += 'originated_rollup' in result ? tx_rollup_origination_size : 0;
+      totalStorage +=
```

Comment:
we should move in this condition inside the loop as discussed

Reply:
ahh ok i didnt realize that the `genesis_commitment_hash` was sr specific